cmd/coreinstance/create_core_instance_operator.go

```diff
@@ -245,6 +268,9 @@ func newCmdCreateCoreInstanceOperator(config *cfg.Config, testClientSet kubernet
 	fs.StringVar(&coreInstanceName, "name", "", "Core instance name (autogenerated if empty)")
 	fs.StringVar(&coreFluentBitDockerImage, "fluent-bit-image", "", "Calyptia core fluent-bit image to use.")
 
+	fs.StringVar(&coreDockerToCloudImage, "image-to-cloud", "", "Calyptia core operator (to-cloud) docker image to use (fully composed docker image).")
```

Comment:
once this is needed just because we need to set the image for testing, i think that we could hide this flag from the client

Reply:
@gabrielbussolo good idea, solved.